### ISSUE TITLE
[BUG]: 重启时显示 `Class Widgets 正在运行`

### ISSUE THREAD

**Issue by xuanxuan1231** [Class-Widgets/Class-Widgets#272]
### 确认检查清单

- [X] 我已阅读以上清单并确认。

### 问题发生的位置

关闭/重启

### 实际结果

重启后显示有另一个进程在运行，仍然需要手动重启

### 重现步骤

自重启

### 应用版本

1.1.7-b5

### 操作系统版本

macOS 12.7.5

### 最后一步

- [X] 我认为上述的描述已经足以详细，以允许开发人员能复现该问题。如果我的 issue 没有按照上述的要求填写，可能会被 **无条件关闭**。

**Comment by RinLit-233-shiroko** [comment]
在Windows端正常，根据反馈，似乎仅在Linux和macOS中出现此问题

**Comment by MCAbleBlank** [comment]
几天前我在Windows上测试版本时出现过这种情况，通过查看任务管理器的进程，有classwidgets的进程在后台运行，但是前台没有窗口

**Comment by MCAbleBlank** [comment]
> 几天前我在Windows上测试版本时出现过这种情况，通过查看任务管理器的进程，有classwidgets的进程在后台运行，但是前台没有窗口

但是没有复现成功

**Comment by xuanxuan1231** [comment]
重启的那部分加一个

```python
share.detach()
```
就行

**Comment by RinLit-233-shiroko** [comment]
> 重启的那部分加一个
> 
> share.detach()
> 就行

hao

**Comment by xuanxuan1231** [comment]
插件广场的重启和主程序的重启没写一起……所以现在插件广场重启还是有问题

**Comment by xuanxuan1231** [comment]
然而只修了一半

**Comment by xuanxuan1231** [comment]
太奇怪了重启好了关闭又坏了
